### PR TITLE
Dev: Remove the Helpers ignore errors in phpstan-baseline.php

### PR DESCRIPTION
**Pull request by jinny8813**
**Description**
See #7731

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Review by datamweb — COMMENTED** [review]
https://github.com/codeigniter4/CodeIgniter4/actions/runs/5758096420/job/15610910416?pr=7777
```console
Error: Function trace() should return int but return statement is missing.
 ------ --------------------------------------------------------------------- 
  Line   system/Helpers/kint_helper.php                                       
 ------ --------------------------------------------------------------------- 
  78     Function trace() should return int but return statement is missing.  
 ------ --------------------------------------------------------------------- 
```
see
https://github.com/kint-php/kint/blob/a700653a77250b122920799b10c94e904c9b78c7/src/Kint.php#L472

File: system/Helpers/kint_helper.php [hunk 1]
```diff
@@ -16,7 +16,9 @@
         /**
          * Prints a Kint debug report and exits.
          *
-         * @param array ...$vars
+         * @param array $vars
+         *
+         * @return void
```
Comment (kenjis) [inline system/Helpers/kint_helper.php:21]:
```suggestion
         * @return void
         * @phpstan-return never
```

File: system/Helpers/kint_helper.php [hunk 2]
```diff
@@ -51,13 +67,23 @@ function d(...$vars)
         /**
          * Provides a backtrace to the current execution point, from Kint.
          */
+        /**
+         * trace function
+         *
+         * @return int
```
Comment (kenjis) [inline system/Helpers/kint_helper.php:73]:
```suggestion
         * @return void
```

File: system/Helpers/kint_helper.php [hunk 3]
```diff
@@ -51,13 +67,23 @@ function d(...$vars)
         /**
          * Provides a backtrace to the current execution point, from Kint.
          */
+        /**
+         * trace function
+         *
+         * @return int|string
```
Comment (kenjis) [inline system/Helpers/kint_helper.php:73]: This method return nothing.
```suggestion
         * @return void
```

Comment (jinny8813) [reply system/Helpers/kint_helper.php:73]: OK. Tks for review

File: system/Helpers/kint_helper.php (hunk 1 above)
Comment (kenjis) [reply system/Helpers/kint_helper.php:21]: Please apply my suggested change.
The true return type is not `void`, but `never`.
See https://phpstan.org/writing-php-code/phpdoc-types#bottom-type
https://www.php.net/manual/en/language.types.never.php

**Comment by kenjis** [comment]
```
Error: Function trace() has no return type specified.
 ------ ------------------------------------------------ 
  Line   system/Helpers/kint_helper.php                  
 ------ ------------------------------------------------ 
  70     Function trace() has no return type specified.  
 ------ ------------------------------------------------ 
```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/5759908201/job/15614892837?pr=7777

**Comment by datamweb** [comment]
@jinny8813 You can run the phpstan analyse before push it on GitHub.
```console
php vendor/bin/phpstan analyse
```
more information [see](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#static-analysis-on-php-code)

**Comment by jinny8813** [comment]
> @jinny8813 You can run the phpstan analyse before push it on GitHub.
> 
> ```
> php vendor/bin/phpstan analyse
> ```
> 
> more information [see](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#static-analysis-on-php-code)

tks for helping me!! it return no error in ./system/Helpers!!

**Review by datamweb — APPROVED** [review]

**Review by kenjis — APPROVED** [review]
LGTM!

**Comment by kenjis** [comment]
@jinny8813 Thank you!

**Pull request merged**